src/Util/Strings.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @file src/Util/Strings.php
+ */
+
+namespace Friendica\Util;
+
+/**
+ * @brief This class contains methods to modify/transform strings.
+ */
+class Strings
```

Comment:
We don't use plural anywhere else in class names and especially not in `Util`, I'm not sure it's relevant to start using it now. Please rename to `Util\String`.

Reply:
Originally I intended on doing that, but String is a reserved word and gave me errors when attempting to create `class String`

Reply:
Util\Arrays.php is where I took my inspiration in using plural

Reply:
In fact I used the word "Objects" as a class for the same reason.

Reply:
Ha, I stand corrected.